pkg/action/list.go

```diff
@@ -19,13 +19,17 @@ package action
 import (
 	"path"
 	"regexp"
+	"strings"
 
 	"k8s.io/apimachinery/pkg/labels"
 
+	"helm.sh/helm/v3/pkg/cli"
 	"helm.sh/helm/v3/pkg/release"
 	"helm.sh/helm/v3/pkg/releaseutil"
 )
 
+var settings = cli.New()
```

Comment:
It's related to: https://github.com/helm/helm/pull/11674#discussion_r1060874323

If getter takes form cfg instead of settings then this can be removed. 

```go
getter := settings.RESTClientGetter()

---->

getter := cfg.RESTClientGetter.(genericclioptions.RESTClientGetter)
```